plugins/woocommerce-admin/client/marketing/overview-multichannel/DiscoverTools/DiscoverTools.tsx

```diff
@@ -45,40 +49,36 @@ const getTabs = ( plugins: Plugin[] ) => {
 	} ) );
 };
 
-const renderPluginCardBodies = ( plugins: Plugin[] ) => {
-	return plugins.map( ( el, idx ) => {
-		return (
-			<Fragment key={ el.product }>
-				<PluginCardBody
-					icon={ <img src={ el.icon } alt={ el.title } /> }
-					name={ el.title }
-					pills={ el.tags.map( ( tag ) => (
-						<Pill key={ tag.slug }>{ tag.name }</Pill>
-					) ) }
-					description={ el.description }
-					button={
-						<Button
-							variant="secondary"
-							href={ getInAppPurchaseUrl( el.url ) }
-							onClick={ () => {
-								recordEvent(
-									'marketing_recommended_extension',
-									{ name: el.title }
-								);
-							} }
-						>
-							{ __( 'Get started', 'woocommerce' ) }
-						</Button>
-					}
-				/>
-				{ idx !== plugins.length - 1 && <CardDivider /> }
-			</Fragment>
-		);
-	} );
-};
-
 export const DiscoverTools = () => {
-	const { isLoading, plugins } = useRecommendedPlugins();
+	const { isLoading, plugins, removeRecommendedPlugin } =
+		useRecommendedPlugins();
+	const [ currentPlugin, setCurrentPlugin ] = useState< string | null >(
+		null
+	);
+	const { installAndActivatePlugins } = useDispatch( PLUGINS_STORE_NAME );
+	const { loadInstalledPluginsAfterActivation } = useInstalledPlugins();
+
+	const installAndActivate = async ( plugin: Plugin ) => {
```

Comment:
A JSDOC comment at the top of this method would be cool

Reply:
Done in [`aac6c8b` (#35542)](https://github.com/woocommerce/woocommerce/pull/35542/commits/aac6c8bffd88ac6f07f752d6ee49eaa5e1a159b7).